collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
+
+Below is an overview of the proposed measures, including references to the European Commission's original legislative proposals:
+
+### Market mechanisms
+
+#### Emission allowances
+
+* [Revision of emissions trading](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0551) (*European Emissions Trading System*, EU ETS)
+* Inclusion of [aviation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0552) and [maritime transport](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0551) in the EU ETS
+* [Extending the EU ETS to road transport and buildings](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0551)
+
+#### Other market mechanisms
+
+* [Carbon offsetting at the border](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0564) (*Carbon Border Adjustment Mechanism*, CBAM)
+* [Revision of the Energy Taxation Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0563) (*Energy Taxation Directive*, ETD)
+
+### Objectives and regulation
+
+#### Commitments to reduce emissions (in sectors not burdened by market mechanisms)
+
+* [Revision of the Effort Sharing Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0555)
+
+#### Energy
+
+* [Revision of the Renewable Energy Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0557) (*Renewable Energy Directive*, RED)
+* [Revision on reducing methane emissions in the energy sector and amending regulation (EU) 2019/942](https://eur-lex.europa.eu/legal-content/cs/TXT/?uri=CELEX%3A52021PC0805)
+* [Revision of the regulation on the internal markets for renewable and natural gas and hydrogen](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=COM%3A2021%3A804%3AFIN)
+* [Revision of the directive on common rules for the internal markets in renewable and natural gas and in hydrogen](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=COM%3A2021%3A803%3AFIN)
+
+#### Energy efficiency
+
+* [Revision of the Energy Efficiency Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0558) (*Energy Efficiency Directive*, EED)
+* [Revision of the Energy Performance of Buildings Directive](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=COM%3A2021%3A802%3AFIN) (*Energy Performance of Buildings Directive*, EPBD)
+
+#### Transportation
+
+<!--* [Stricter emission regulations for cars and vans](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556)-->
```

Comment:
All the eur-lex.europa.eu are easy to update by changing them from https://eur-lex.europa.eu/legal-content/CS to https://eur-lex.europa.eu/legal-content/EN.

However, I tried to use the English URL for this link but 
https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556 only appears in French.

Reply:
would just using the PDF be sufficient? I think that for whatever reason the HTML just doesn't exist (in any case I tried to find it and could not)

Reply:
Yes, I think the PDF should be sufficient but wanted to make sure I did not look into the wrong place for the html version.